scripts/gatk4_selectvariants_test.pbs

```diff
@@ -0,0 +1,56 @@
+#! /bin/bash
+
+#########################################################
+#
+# Platform: NCI Gadi HPC
+# Usage: qsub gatk4_selectvariants.pbs
+# Version: 1.0
+#
+# For more details see: https://github.com/Sydney-Informatics-Hub/Somatic-ShortV
+#
+# If you use this script towards a publication, support us by citing:
+#
+# Suggest citation:
+# Sydney Informatics Hub, Core Research Facilities, University of Sydney,
+# 2021, The Sydney Informatics Hub Bioinformatics Repository, <date accessed>,
+# https://github.com/Sydney-Informatics-Hub/Germline-ShortV
+#
+# Please acknowledge the Sydney Informatics Hub and the facilities:
+#
+# Suggested acknowledgement:
+# The authors acknowledge the technical assistance provided by the Sydney
+# Informatics Hub, a Core Research Facility of the University of Sydney
+# and the Australian BioCommons which is enabled by NCRIS via Bioplatforms
+# Australia. The authors acknowledge the use of the National Computational
+# Infrastructure (NCI) supported by the Australian Government.
+#
+#########################################################
+
+#PBS -P er01
+#PBS -N selectvariants
+#PBS -l walltime=04:00:00,ncpus=48,mem=192GB,wd
+#PBS -q normal
+#PBS -W umask=022
+#PBS -l storage=scratch/er01+gdata/er01
+#PBS -o ./Logs/selectvariants.o
```

Comment:
Has this `./Logs` directory already been made? It will not be automatically made for users upon execution so you need to place the logs somewhere else.

Reply:
ok